src/index.ts

```diff
@@ -280,6 +283,34 @@ class Monika extends Command {
           em.emit('SANITIZED_CONFIG', sanitizedProbe)
         }
 
+        // run TLS checker
+        if (config?.certificate && config.certificate.domains.length > 0) {
+          config.certificate?.domains.forEach((domain) => {
+            // TODO: Remove probe below
+            // probe is used because probe detail is needed to save the notification log
+            const probe = sanitizedProbe[0]
```

Comment:
Since this TLS expiry notification does not need probe, can't we just create a dummy probe here?

Reply:
I updated here [f36e459](https://github.com/hyperjumptech/monika/pull/315/commits/f36e4590d509bc4692e595ca383b797fb659f526)